incubator/harbor/templates/ingress/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "{{ template "harbor.fullname" . }}-ingress"
+  labels:
+{{ include "helm.labels" . | indent 4 }}
+  annotations:
```

Comment:
Why are these hard-coded? With what ingress controller are they supposed to work? FWIW, nginx-ingress has changed annotations recently. I'd suggest you make those configurable.